laurent22/joplin
# All: Allow passing custom data to Stripe

Track the origin of a user payment (desktop, mobile) and pass it to Stripe. The goal is to get a better sense of where most payments come from.

### How to test
Follow instructions in `stripe.md` then set a source with `localStorage.setItem('source', 'desktop')` and proceed with a checkout. The source will appear in Stripe's dashboard under `Customers`.
![customer-dashboard](https://user-images.githubusercontent.com/32807437/226026785-840e79e9-de12-49d3-933b-27ae3452da95.png)
